modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionContainer/ActionContainer.tsx

```diff
@@ -184,6 +186,17 @@ export function ActionContainer({
 		values.objectActionExecutorKey,
 	]);
 
+	useEffect(() => {
+		const makeFetch = async () => {
+			const objectDefinition = await API.getObjectDefinitionByExternalReferenceCode(
+				objectDefinitionExternalReferenceCode
+			);
+			setCreationLanguageId(objectDefinition.defaultLanguageId);
```

Comment:
@rebsilva Please, add a blank line between the definition of `objectDefinition` and its usage. Thanks :)

Reply:
Will do, thanks!